traceapp/app.go

```diff
@@ -39,6 +39,8 @@ type App struct {
 
 	tmplLock sync.Mutex
 	tmpls    map[string]*htmpl.Template
+
+	Log bool
```

Comment:
I think it would make sense to have this be a `*log.Logger` instead. See how we do this for ChunkedCollector for example: https://github.com/sourcegraph/appdash/blob/master/collector.go#L148